lib/cpus/aarch64/neoverse_n1.S

```diff
@@ -1,24 +1,24 @@
 /*
- * Copyright (c) 2017-2018, ARM Limited and Contributors. All rights reserved.
+ * Copyright (c) 2017-2019, ARM Limited and Contributors. All rights reserved.
  *
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
 #include <arch.h>
 #include <asm_macros.S>
-#include <cortex_ares.h>
+#include <neoverse_n1.h>
 #include <cpuamu.h>
 #include <cpu_macros.S>
 
 /* --------------------------------------------------
- * Errata Workaround for Cortex-Ares Errata
- * This applies to revision r0p0 and r1p0 of Cortex-Ares.
+ * Errata Workaround for Neoverse N1 Errata
+ * This applies to revision r0p0 and r1p0 of Neoverse N1.
  * Inputs:
  * x0: variant[4:7] and revision[0:3] of current cpu.
  * Shall clobber: x0-x17
  * --------------------------------------------------
  */
-func errata_ares_1043202_wa
+func errata_n1_1043202_wa
```

Comment:
Just a minor question. Should this be errata_neoverse_n1_1043202_wa ?

Reply:
Hello @thomas-arm ,

If you see in other similar files, e.g. cortex_a53.S, the errata functions and flags use the shortened name of the core, for example "errata_a53_...._wa" ("cortex" is missing). So we applied the same concept here to make the function name shorter and more readable. Makes sense?

Reply:
Ok, that is fine. Thank you. Just wanted to ensure that name qualifications are complete just in case there is a another n1 cpu of a different category.

Reply:
> just in case there is a another n1 cpu of a different category
That's a valid point but, theoretically, that shouldn't happen since Cortex family uses A/R/M and Neoverse family uses N/E. Otherwise we'll indeed have to do more renamings in the future :)